### DISCUSSION TITLE
CMS Editing: Validierung

### DISCUSSION THREAD

**Discussion by thomaskofler** [e-netze/webgis-community#49] [General]
Hallo,

ich habe in einer Editierumgebung bei einem Attribut eine Validierung angegeben - wird das Attribut falsch ausgefüllt kommt ein Fenster mit Benachrichtigung:

![grafik](https://user-images.githubusercontent.com/119412172/225539098-bd06fbb5-7ab6-4a5d-b301-2304c1c495ee.png)

Wenn man "Attribute bearbeiten" über die Abfrage macht, dann eine Falscheingabe macht und das Benachrichtigungsfenster schließt, dann springt man wieder ganz aus dem Editierbereich raus, wäre es möglich, dass man nach dem Schließen des Fensters in dem Editierbereich bleibt, damit man eine korrekte Eingabe machen kann?
![grafik](https://user-images.githubusercontent.com/119412172/225540076-07c0c06c-1a47-4e57-b4e1-1adec92fd006.png)

LG Thomas

**Comment by gis-eni** [comment]
Die Fehler bei serverseitiger Validierung werden einem einem Popup Dialog dargestellt. Da nur ein Popup Dialog gleichzeitig geöffnet sein kann, schließt diese den Sachdateneditor. Um das zu lösen wäre einiger Umbau notwendig und das kommt erst mit der "neuen" Editierumgebnung (``webgis 6.23.xxxx``).

Was du für einfache Validierungen (wie zB ``required``) probieren kannst ist clientseitige Validierung zu verwenden:

![image](https://user-images.githubusercontent.com/62870568/229999757-58880e5b-063b-41a8-abe2-803d55408d63.png)

Dabei werden die Werte schon vor dem Absenden zum Server überprürgt:

![image](https://user-images.githubusercontent.com/62870568/229999928-3c63c309-342f-45ce-8b65-82dc7ba68427.png)
